tests/functional/lib.sh

```diff
@@ -19,14 +19,20 @@ _sudo() {
 
 HEKETI_PID=
 start_heketi() {
+    local config_filename=$1
     ( cd "$HEKETI_SERVER_BUILD_DIR" && make && cp heketi "$HEKETI_SERVER" )
     if [ $? -ne 0 ] ; then
         fail "Unable to build Heketi"
     fi
 
+    if [ ! -f config/"${config_filename}" ]
+    then
+        config_filename="heketi.json"
+    fi
+
     # Start server
     rm -f heketi.db > /dev/null 2>&1
-    $HEKETI_SERVER --config=config/heketi.json &
+    $HEKETI_SERVER --config=config/${config_filename} &
```

Comment:
All the changes to this function look fine to me. It makes the function more general.